### PR TITLE
[#2] Added makefile

### LINKED ISSUE #2: Addition of makefile
## Expected Behavior
Be able to install/uninstall all tgit scripts only with a make command.

## Context (Environment)
Currently in the project, all the scripts need to be copied manually to /usr/local. The same is expected for removing, that means, manually deleting the script files.

## Detailed Description
Add a makefile in the tgit repo, so all the scripts contained in it can be installed and removed automatically.

The makefile is expected to install/uninstall the following scripts:
- tgit
- dmenu_tgit
- tgit_status

## Possible Implementation
Implementation of a makefile is enough, following the pattern already seen on other tiny tools installation method, like [tnotes](https://github.com/TinyToolSH/tnotes/blob/main/Makefile), for example.

### PR DESCRIPTION
**Pull request by gbgabo**
## Description
Added makefile for installation and uninstallation of tgit applications. Updated README mentioning installation method.

## Related Issue
Closes #2 

## How Has This Been Tested?
- Manually uninstalled tgit scripts
- Ran make install command
- Called each tgit script on terminal
- Ran make uninstall command

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,15 +15,26 @@ $ dmenu_tgit
...
 
 ## dependencies:
+
 - git
 - find
 - awk
 
+## Instalation
+
+To install `tmenu` you can edit the `Makefile` to match your local setup (`tmenu` is installed into the `/usr/local/bin` by default).
```
Comment (calebe94) [inline README.md]: The name of the package is wrong.
It should be `tgit` instead.

**Review by calebe94 — CHANGES_REQUESTED** [review]
There are some changes pending...

File: README.md (hunk 1 above)
Comment (gbgabo) [reply README.md]: silly mistake of mine 😬
Just fixed it!

**Review by calebe94 — APPROVED** [review]
Everything seems OK to me!

**Pull request merged**